src/mapping/RadialGeoMultiscaleMapping.hpp

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include <vector>
+#include "logging/Logger.hpp"
+#include "mapping/Mapping.hpp"
+
+namespace precice {
+namespace mapping {
+
+/// Geometric multiscale mapping in axial direction
+class RadialGeoMultiscaleMapping : public Mapping {
+public:
+  /**
+   * @brief Geometric multiscale nature of the mapping (spread or collect).
+   *
+   * A geometric multiscale mapping can either go from the 1D to the 2D/3D solver. Then, we call it "spread".
+   * Or from the 2D/3D to the 1D solver, which we call "collect".
+   */
+  enum MultiscaleType {
+    SPREAD,
+    COLLECT
+  };
+  enum MultiscaleAxis {
+    X,
+    Y,
+    Z
+  };
```

Comment:
Could be moved to a common header to avoid the duplication?

Reply:
The goal would be to have a geometric multiscale base class that implements this. We didn't yet do it because we only have two kinds of geometric multiscale mappings, but a base class would also make it easier to implement all kinds of geometric multiscale specifications. But it would have to be investigated what commonalities could be put in a base class amd what would have to be implemented for each child (e.g., `radius` needed for axial but not `radial`).

Reply:
I will resolve this when creating a base class, in the context of #1668.